Sources/Plasma/NucleusLib/inc/plPipeline.h

```diff
@@ -354,7 +359,11 @@ class plPipeline : public plCreatable
     plDisplayMode fDesktopParams;
 
     virtual size_t GetViewStackSize() const = 0;
-    
+
+#ifdef HS_BUILD_FOR_MACOS
+    virtual CALayer* GetRenderLayer() { return nullptr; }
```

Comment:
```suggestion
    virtual CALayer* GetRenderLayer() const { return nullptr; }
```

Looks like a simple getter that should be a const member.